cluster/server.go

```diff
@@ -34,6 +34,7 @@ type Server struct {
 	Metadata map[string]string `json:"metadata"`
 	Frontend bool              `json:"frontend"`
 	Hostname string            `json:"hostname"`
+	Loopback bool              `json:"loopback"`
```

Comment:
Have you considered naming the field `AllowLoopback` instead? I'm thinking that having it called only `Loopback` could give the impression that it'd loopback all calls, but maybe I'm overthinking it!